examples/guestbook/README.md

```diff
@@ -38,20 +38,17 @@ This example shows how to build a simple, multi-tier web application using Kuber
 
 **Table of Contents**
 
-  - [Step Zero: Prerequisites](#step-zero-prerequisites)
+  - [Prerequisites](#prerequisites)
```

Comment:
Use auto-generated table of contents instead. Add the following tag and then run `hack/update-generated-docs.sh`.
```
<!-- BEGIN MUNGE: GENERATED_TOC -->
<!-- END MUNGE: GENERATED_TOC -->
```